### PR TITLE
Right nav stickiness

### PR DESCRIPTION
**Pull request by codemonkey800**
## Description

Fixes the stickiness for the right nav in the plugin and markdown pages so that the right nav sticks only when its `50px` from the top of the screen after scrolling.

### Plugin Page

https://right-nav-stickiness-frontend.dev.imaging.cziscience.com/plugins/napari-btrack-reader

https://user-images.githubusercontent.com/2176050/123850261-a64f2780-d8ce-11eb-9244-c44dcee5bae2.mp4

### Markdown Pages

https://right-nav-stickiness-frontend.dev.imaging.cziscience.com/about

https://user-images.githubusercontent.com/2176050/123850254-a3543700-d8ce-11eb-9209-086edf7f5a9f.mp4

### CODE REVIEW

File: client/src/global.scss [hunk 1]
```diff
@@ -1,13 +1,6 @@
 @import '@/utils/styles';
 
 html {
-  /*
-    Enable scrollbar for the entire page by default. This is to prevent
-    layout shift when switching between pages that have content long enough /
-    not long enough for a scrollbar.
-  */
-  @apply overflow-y-scroll;
```
Comment (codemonkey800) [inline client/src/global.scss:9]: Removing this is required for `position: sticky` to work. I think all of our pages have enough content to be scrollable, so I don't think we have to worry about this

**Review by liaprins-czi — APPROVED** [review]
Awesome work! Looks great to me!

File: client/src/components/PluginDetails/PluginDetails.tsx [hunk 2]
```diff
@@ -87,19 +87,22 @@ function PluginRightColumn() {
...
-        markdown={plugin.description}
-        onClick={(section) => {
-          plausible('Description Nav', {
-            section,
-            plugin: plugin.name,
-          });
-        }}
-      />
+      <div className="sticky top-12">
+        {/*  Keep button on screen when scrolling on 2xl. */}
```
Comment (justinelarsen) [inline client/src/components/PluginDetails/PluginDetails.tsx:91]: Update comment to keep TOC sticky instead?

File: client/src/global.scss (hunk 1 above)
Comment (justinelarsen) [reply client/src/global.scss:9]: What about the Contact page? That one is pretty short.

**Review by justinelarsen — COMMENTED** [review]
Awesome work! I just added a couple comments.

File: client/src/global.scss (hunk 1 above)
Comment (codemonkey800) [reply client/src/global.scss:9]: Good point, the page isn't scrollable as you mention. Still, I don't think we have to worry too much about. It's a super minor thing that only a handful of people will experience, and even then, they might not even notice it that much.

Comment (justinelarsen) [reply client/src/global.scss:9]: Sounds good. Let's keep it in mind.

**Review by justinelarsen — APPROVED** [review]

**Pull request merged**